nautobot/extras/models/change_logging.py

```diff
@@ -74,6 +74,7 @@ class ObjectChange(BaseModel):
         blank=True,
         null=True,
     )
+    # todoindex:
```

Comment:
```
extras/views.py-1419-            ObjectChange.objects.restrict(request.user, "view")
extras/views.py-1420-            .prefetch_related("user", "changed_object_type")
extras/views.py:1421:            .filter(
extras/views.py-1422-                Q(changed_object_type=content_type, changed_object_id=obj.pk)
extras/views.py-1423-                | Q(related_object_type=content_type, related_object_id=obj.pk)
```

Reply:
I know we want to avoid the scope creep on this PR to include "index togethers", but @glennmatthews @jathanism, would this be a better case of adding into the `Meta` below?

```py
models.Index(
    name="extras_objectchange_related_idx",
    fields=["related_object_id", "related_object_type_id"],
),
```